core/src/test/java/org/testcontainers/junit/GenericContainerRuleTest.java

```diff
@@ -204,6 +215,12 @@ public void customClasspathResourceMappingTest() throws IOException {
     }
 
     @Test
+    public void customClasspathResourceMappingWithSelinuxTest() throws IOException {
```

Comment:
I'm not an SELinux expert, but maybe this might help?
https://www.cyberciti.biz/faq/howto-display-selinux-security-context-using-ls/